src/data_store/data_store_multi_images.cpp

```diff
@@ -68,24 +80,22 @@ void data_store_multi_images::setup() {
 }
 
 void data_store_multi_images::get_file_sizes() {
-  std::vector<Triple> my_file_sizes(m_my_global_indices.size()*m_num_img_srcs);
-  std::pair<std::vector<std::string>, int> sample;
-  size_t cur_offset = 0;
-  data_reader_multi_images *reader = dynamic_cast<data_reader_multi_images*>(m_reader);
+  std::vector<Triple> my_file_sizes(m_my_datastore_indices.size()*m_num_img_srcs);
+  if (m_master) std::cerr << "STARTING data_store_multi_images::get_file_sizes for " << my_file_sizes.size() << " files\n";
```

Comment:
For a message like this, we should use overriden get_name() for each derived class instead of hard-coded name. Currently, data_store_triplet inherits this function and prints out "data_store_multi_images::get_file_sizes for ... ". So, we also need to add set_name("data_store_xxx") to setup()